Dockerfile

```diff
@@ -18,8 +18,9 @@ COPY /privbayes/ /SDGym/privbayes
 WORKDIR /SDGym
 
 # Install project
-RUN make install-all compile
-RUN pip install -U numpy==1.20
+RUN pip install ydata-synthetic==0.6.1
+RUN pip install .[gretel] --no-binary pomegranate
+RUN make compile
```

Comment:
Had a question - was there an issue with running `make install-all compile`?

Reply:
Yes. The problem is that when running through `make` I did not have the `--no-binary pomegranate` flag, without which the installation failed when building the docker image.
On the other hand, outside of the docker it is not needed, so adding it to the `Makefile` target seemed unnecessary.